dev/config-mariadb.yaml

```diff
@@ -12,7 +11,10 @@ security:
           - create
         scopes:
           - Project
-
+  authentication:
+    providers:
+      - kind: "database"
```

Comment:
I think `native` would require less explanation. Specially when you are reading the config like that: here my list of auth providers: `database`.

That doesn't sound a great name for an auth provider :)

Reply:
ok for native :)